### DISCUSSION TITLE
Fire Max 11 Support

### DISCUSSION THREAD

**Discussion by DrujVijfde** [weto91/kindle-cover-fixer#19] [General]
Does this support the Fire Max 11? I continue to see the message to connect my Kindle, which I have connected via USB/file transfer, but nothing.

Thanks!

**Comment by weto91** [comment]
Sorry but is not tested with the Fire devices, only with the e-ink devices as peperwhite, scribe.. you can see the entire compatible device list from: https://github.com/weto91/kindle-cover-fixer
